examples/signeduploads/sampleapp/views.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+import json
+import time
+from .forms import ImgForm
+from pyuploadcare import generate_secure_signature
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+
+
+def index(request):
+    imgForm = ImgForm()
+    return render(request, 'sampleapp/index.html', {'form': imgForm})
+
+
+@csrf_exempt
+def release_signature(request):
```

Comment:
1. The idea is that server releases signature by request from logged-in client.
2. Agreed. I just didn't know you could do the app separate from the project :)